### PR TITLE
Igpu transfer

### PR DESCRIPTION
**Pull request by hyesoon**

### CODE REVIEW

File: src/frontend.cc [hunk 1]
```diff
@@ -372,11 +382,11 @@ FRONTEND_MODE frontend_c::process_ifetch(unsigned int tid,
...
-      } else if (KNOB(KNOB_LARGE_CORE_TYPE)->getValue() == "igpu") {
-        trace_info_igpu_s *prev_trace_info =
-          static_cast<trace_info_igpu_s *>(thread->m_prev_trace_info);
-        fetch_data->m_MT_scheduler.m_next_fetch_addr =
-          prev_trace_info->m_instruction_addr;
+      // } else if (KNOB(KNOB_LARGE_CORE_TYPE)->getValue() == "igpu") {
+        // trace_info_igpu_s *prev_trace_info =
+          // static_cast<trace_info_igpu_s *>(thread->m_prev_trace_info);
+        // fetch_data->m_MT_scheduler.m_next_fetch_addr =
+          // prev_trace_info->m_instruction_addr;
```
Comment (hyojongk) [inline src/frontend.cc]: delete these lines?

File: macsimComponent.cpp [hunk 2]
```diff
@@ -60,7 +60,18 @@ macsimComponent::macsimComponent(ComponentId_t id, Params& params)
     m_clock_freq,
     new Clock::Handler<macsimComponent>(this, &macsimComponent::ticReceived));
 
-  m_ptx_core = params.find<bool>("ptx_core", 0);
+  m_acc_core = params.find<bool>("ptx_core", 0);
```
Comment (hyojongk) [inline macsimComponent.cpp]: delete this line?

File: src/memory.h [hunk 3]
```diff
@@ -303,8 +303,9 @@ class dcu_c
   int m_line_size; /**< cache line size */
   int m_banks; /**< number of cache banks */
   int m_latency; /**< cache access latency */
-  bool m_ptx_sim; /**< gpu cache */
+  bool m_acc_sim; /**< gpu cache */
   bool m_igpu_sim; /**< intel gpu cache */
+  bool m_ptx_sim; /**< gpu cache */
```
Comment (hyojongk) [inline src/memory.h:308]: aren't we replacing m_ptx_sim with m_acc_sim? then delete this?
or if m_acc_sim is equal to (m_ptx_sim || m_igpu_sim) then, can we change line 1492 or 1524 to if (m_acc_sim), not if (m_acc_sim || m_igpu_sim)?

**Review by hyojongk — CHANGES_REQUESTED** [review]

File: src/memory.h (hunk 3 above)
Comment (hyesoon) [reply src/memory.h:308]: m_acc_sim = (m_ptx_sim || m_igpu_sim)

File: macsimComponent.cpp (hunk 2 above)
Comment (hyesoon) [reply macsimComponent.cpp]: actually, I'm not sure what I need to do.

**Review by hyojongk — DISMISSED** [review]

**Review by hyojongk — APPROVED** [review]

**Pull request merged**